industrial_ci/src/docker.env

```diff
@@ -15,6 +15,7 @@ CLANG_TIDY_ARGS
 CLANG_TIDY_JOBS
 CMAKE_ARGS
 CODE_COVERAGE
+CODE_COVERAGE_IGNORE_PACKAGES
```

Comment:
Use `CODE_COVERAGE_IGNORE`, by default this will be `"*/test/* */setup.py"`, if it is a relative path to a folder within the repository, this will be something like `"<package_name> */test/* */setup.py"`

Reply:
So if `CODE_COVERAGE_IGNORE="pkg1 */test/* */setup.py`

```
  local -a ignored
  ici_parse_env_array ignored CODE_COVERAGE_IGNORE

  for ign in "${ignored[@]}"; do
      lcov --remove coverage.info "$ign" \
           --output-file coverage.info | grep -ve "^removing"
  done
```
Would not work because it would have to be
`CODE_COVERAGE_IGNORE="*/pkg1/* */test/* */setup.py`

Right?